etna/records/models.py

```diff
@@ -242,6 +240,18 @@ def description(self) -> str:
             return format_description_markup(raw)
         return ""
 
+    @property
+    def raw_content(self) -> str:
+        for item in self.get("source.content.items", ()):
+            return item.get("value", "")
+        return ""
+
+    @cached_property
+    def content(self) -> str:
+        if raw := self.raw_content:
+            return format_description_markup(raw)
```

Comment:
This implementation caters for links in response. Its not same as [previous implementation](https://github.com/nationalarchives/ds-wagtail/blob/36f31a94d555fba18c8275572d9224e4c340998b/templates/search/blocks/featured-search__interpretive-card.html#L24) `{{ record|interpretive_detail:"content"|safe|striptags|truncatewords:40 }}` 
Are there links in the response?

Reply:
Thanks Allan. This was an oversight on my part. I have swapped the `format_description_mark()` treatment here for a bleach-based `strip_tags()` function, which should be safer than the django tag (as outlined here: https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#striptags).

Fixed in d6720a6